pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -471,13 +475,14 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *api.Ser
 }
 
 func (lbaas *LbaasV2) UpdateLoadBalancer(clusterName string, service *api.Service, hosts []string) error {
+	var err error
+	var listenersList []listeners.Listener
+	var poolsList []v2_pools.Pool
```

Comment:
Style/defensive coding nit: I'd prefer we created variables closer to where they were first used.